### ISSUE TITLE
Thought on Incident's Subscribers list UX

### ISSUE THREAD

**Issue by martialblog** [Icinga/icinga-notifications-web#256]
Hi, 

whilst testing I noticed that the behavior of the Incident's Subscribers list confused me at first.

By that I mean, when clicking on an Item in the Subscribers I thought I opened a tab to manage the Subscriber (not the Contact) - since I'm the admin I though: if I click on delete I remove the subscriber... alas I didn't, I removed the contact with no confirmation.

Here a very professional image of what I mean:

![subs](https://github.com/user-attachments/assets/f3cdabba-b991-4349-88a6-f236cf56f33e)

**Comment by nilmerg** [comment]
Yeah, well. Suppose we have a contact detail some day, this isn't an issue anymore. So I won't make one out this now either. 😅

**Comment by martialblog** [comment]
A very small improvement to avoid confusion might just be renaming the button "Delete Contact" instead of "Delete"?
